BuildLinux.sh

```diff
@@ -1,10 +1,8 @@
 #!/bin/bash
 
-export ROOT=`pwd`
-export NCORES=`nproc --all`
+export ROOT=$(dirname $(readlink -f ${0}))
+export NCORES=$(nproc --all)
 export CMAKE_BUILD_PARALLEL_LEVEL=${NCORES}
```

Comment:
the parallel level is not required with ninja bc it always scales to the maximum parallelization

Reply:
It is needed for advanced caching